### PR TITLE
Branch dg and PPP elysia

### PR DESCRIPTION
**Pull request by ElysiaTanZY**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -153,55 +153,157 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
-
Step 1. The user launches the application for the first time. The `SchoolworkTracker` will be initialized with the initial state, which is an empty list.
+[.text-justify]
+It utilizes the existing `Assignment` stored in the `SchoolworkTracker` and implements `ScheduleList` which represents the generated schedule.
 
-Step 2. The user executes the add assignment command [`(st)add t/CS2103 tP d/2020-03-27 12:00 e/5`] to add the following assignments to the Schoolwork Tracker.
+[.text-justify]
+As seen from Fig 3 below, `ScheduleList` consists of `Day` objects and each `Day` has 3 components. `totalAllocatedHours` represents the total number of hours allocated to the `Day`.
+`dueAssignment` refers to an assignment that is due on the `Day` and `allocatedAssignment` represents an assignment that has been scheduled to be done on that `Day`. The latter also has an additional field `allocatedHours` which correspond to the suggested amount of time
+that should be spent on the assignment on that `Day`.
```
Comment (Vanessa1098) [inline docs/DeveloperGuide.adoc:184]: how come this is on a new line?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -153,55 +153,157 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
 
-image::stPart2.png[]
+[.text-justify]
+The specific workings of the schedule command will be explained in the paragraphs below.
 
-image::stPart3.png[]
+[.text-justify]
+As seen from Fig 5 below, arguments from the input command is first parsed using the `ScheduleCommandParser` which converts
+the string variable into an integer and then passes it to `ScheduleCommand` for use later on and represents the `NUM_DAYS` queried. If the input argument is invalid, a `ParseException`
+is thrown and the user will be notified of the proper command usage.
```
Comment (Vanessa1098) [inline docs/DeveloperGuide.adoc:213]: this also haha

**Review by Vanessa1098 — COMMENTED** [review]
Looks good!

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (ElysiaTanZY) [reply docs/DeveloperGuide.adoc:184]: Its not i think is just Github saying i added this line O: HAHA

Comment (ElysiaTanZY) [reply docs/DeveloperGuide.adoc:184]: ohh wait erm when converted to pdf its not but i just separated it from the above line so its easier to read on intelliJ HAHA

**Pull request merged**